assets/app/scripts/directives/logViewer.js

```diff
@@ -64,6 +64,32 @@ angular.module('openshiftConsole')
               logLinks.scrollTop();
             };
 
+            var bufferedLines = [];
+            // Display all buffered lines.
+            var update = function() {
+              var elements = [];
+              angular.forEach(bufferedLines, function(line) {
+                // Manipulate the DOM directly for better performance displaying large log files.
+                var logLineDiv = $('<div row class="log-line"/>');
```

Comment:
For a hotspot like this, it's probably worth using DOM methods directly instead of jquery text parsing